src/components/Form/TextArea.tsx

```diff
@@ -43,15 +43,20 @@ function TextArea({
   placeholder,
   onFocus,
   onBlur,
+  value,
   ...otherProps
 }: TextAreaProps) {
   const { disabled } = otherProps
   variant = disabled ? 'disabled' : variant
   const isDisabled = variant === 'disabled'
   const { input, text } = inputVariants[variant]
   const [focused, setFocused] = useState(false)
-  const textareaRef = useRef<HTMLTextAreaElement>(null)
-  const isLabelScalded = !label || focused || textareaRef.current?.value.length
+  const textRef = useRef<HTMLTextAreaElement>(null)
+  const isLabelScalded =
```

Comment:
```js
 const isLabelScalded = !label || focused  || textareaRef.current?.value?.length || otherProps?.value?.toLocaleString()?.length
```

I recommend don't destructure the value and add some optional chaining to avoid issues